README.md

```diff
@@ -251,13 +256,18 @@ Options:
           Defaults to 0 (auto).
 
       --timeout <TIMEOUT>
-          Tool timeout in seconds
+          Tool timeout in seconds.
 
-          Defaults to no timeout
+          Defaults to no timeout.
 
       --log-level <LOG_LEVEL>
           [possible values: trace, debug, info, warn, error, off]
 
+      --on-missing-tool-binary <ON_MISSING_TOOL_BINARY>
+          What to do when the binary of a tool cannot be found
+
+          [possible values: allow, continue, fail-fast]
```

Comment:
The `allow` might be better if it was named `ignore`

`continue` could be `fail`, having fail and fail-fast would make sense maybe

Reply:
Aight